src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -12,11 +12,7 @@
 
 import seedu.address.logic.commands.AddCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
-import seedu.address.model.person.Address;
-import seedu.address.model.person.Email;
-import seedu.address.model.person.Name;
-import seedu.address.model.person.Person;
-import seedu.address.model.person.Phone;
+import seedu.address.model.person.*;
```

Comment:
I think it would be better to keep the single import statements rather than use a wildcard import.